types/validator.yaml

```diff
@@ -38,6 +38,10 @@ schemas:
     properties:
       validator_pubkey:
         $ref: './misc.yaml#/schemas/PublicKey'
+      aggregator_modulo:
+        allOf:
+          - $ref: './misc.yaml#/schemas/Uint64'
+          - description: "Modulo used in [spec](https://github.com/ethereum/eth2.0-specs/blob/v0.9.4/specs/validator/0_beacon-chain-validator.md#aggregation-selection) to determine if validator is aggregator"
```

Comment:
I had to chase down exactly what was being passed around here.

Might make sense to put a clarifying comment, that the validator can then locally calculate `is_aggregator` (aka `bytes_to_int(hash(slot_signature)[0:8]) % aggregator_modulo == 0`)